radio/src/dataconstants.h

```diff
@@ -243,7 +243,7 @@ enum UartModes {
 
 #define LEN_SWITCH_NAME    3
 #define LEN_ANA_NAME       3
-#define LEN_MODEL_FILENAME 16
+#define LEN_MODEL_FILENAME LEN_MODEL_NAME+1
```

Comment:
@HThuren What was the rationale for this change? It was previously hard-coded to always be 16, and not change depending on the radio family, which it now will.

Reply:
I had this in written in review too but didn't click submit apparently.

I'm not quite understanding the correlation between the model name and why this should effect the file name length. The filename is almost always going to be 11 - 12 chars (so 13 for buffer should be safe. modelXX.yml or modelXXX.yml if you have a 100+ models, 16=extra safe. Right now if this is compiled on a small LCD screen LEN_MODEL_FILENAME will be 11 chars, which is too small.

What was the compile warning before in #2243. I feel these two defines don't really have anything to do with each other could there have been another issue where it was using the incorrect define somewhere?

Reply:
As far as I can see, `LEN_MODEL_FILENAME` is only used for targets using the categories/labels, so maybe this is misleading.

Reply:
My understanding was, combined with the changes to function parameter names, that the model file names are going to be the model names.